app/styles/modules/_password-row.scss

```diff
@@ -6,7 +6,12 @@
   position: relative;
 
   &.input-row .password {
-    padding-right: 75px;
+    html[dir=ltr] & {
```

Comment:
If you leave it as default, when RTL Locales the padding will be 75px from right and 75px from left, just like this:

![2016-05-24 22-59-45](https://cloud.githubusercontent.com/assets/9789109/15519758/a29288a0-220b-11e6-917e-36c0cf1c99b5.png)
so i forced the right padding only in LTR Locales and the left padding only in the RTL locales.